src/router.js

```diff
@@ -0,0 +1,21 @@
+const { getRoot, getFromPublic, getAutocomplete } = require('./handlers');
+
+const router = (req, res) => {
+  const endPoint = req.url;
+  const { method } = req;
+
+  if (endPoint === '/') {
+    getRoot(res);
+  } else if (endPoint.includes('/autocomplete') && method === 'GET') {
+    getAutocomplete(req, res, endPoint);
+  } else if (endPoint.indexOf('/') !== -1) {
+    getFromPublic(res, endPoint);
+  } else {
+    res.writeHead(404, { 'Content-Type': 'text/html' });
+    res.end('<h1>Error, Page Is Not Found!!!</h1>');
+  }
+};
+
+if (typeof module !== 'undefined') {
```

Comment:
I believe your environment here is node and module is defined in node, so there is no need to do this check. 

We used to do it in the browser since we needed to export some modules for jest testing meanwhile we didn't want the browser to throw an error since module is not defined in the browser environment.